.github/workflows/public_app_release.yml

```diff
@@ -0,0 +1,26 @@
+name: Trigger Public Release tasks from CirrusCI
+
+on: workflow_dispatch
+permissions: 
+  checks: write
+
+jobs:
+  appstore_public:
```

Comment:
I think we want to split this in two workflows?
one for beta 
one for appstore

Reply:
Do we? I'm not sure what's the case here. Do we usually run these two together or rather trigger them separately?

Reply:
anyway splitted

Reply:
separetly

Reply:
We do have an automation to build appstore public when pushed on `release/cycle-XXX` branch, but we can leave it for now